docs/README.md

```diff
@@ -269,6 +270,16 @@ Endpoints define the entry points of Couper. The mandatory *label* defines the p
 |[**`access_control`**](#access_control_attribute)|sets predefined `access_control` for `endpoint`|
 |[**`backend`**](#backend_block) block |configures connection to a local/remote backend service for `endpoint`|
 
+#### Path parameter
```

Comment:
Missing docu for `beresp.path_param.*`

Reply:
Which other value than `req.path_param` would you expect for `beresp.path_param` ? The user can access `req.path_param` within the same context.